src/test/java/com/mentoring/framework/utils/EmailUtils.java

```diff
@@ -25,12 +25,28 @@
     private static final String USERNAME = "learn.ta.2019@gmail.com";
     private static final String PASSWORD = "Test1234!";
 
-    public static List<String> checkEmailsForIpChanges() {
-        return getListOfEmailContent(HOST, MAIL_STORE_TYPE, USERNAME, PASSWORD);
+    public static String checkLastEmailForIpChanges() {
```

Comment:
This could be private.

Reply:
Public: if we would like to use the content of the last e-mail for any other thing.
But it is unnecessary then we should remove the method and call getLastEmailContent directly.
I think private method is meaningless.